app/src/main/java/org/oppia/android/app/settings/profile/ProfileListFragmentPresenter.kt

```diff
@@ -53,7 +53,7 @@ class ProfileListFragmentPresenter @Inject constructor(
 
   private fun createRecyclerViewAdapter(): BindableAdapter<Profile> {
     return BindableAdapter.SingleTypeBuilder
-      .newBuilder<Profile>()
+      .Factory(fragment).create<Profile>()
```

Comment:
Here & elsewhere: Since ``Factory`` is injectable, we should be injecting it, not constructing it.

Reply:
Noted @BenHenning will make the changes, Will check why i missed this while refactoring and AS didn't show error while building the code.